lib/buildkite.sh

```diff
@@ -236,6 +241,44 @@ EOF
     | @tsv' <<< "${config_json}")
 }
 
+##
+## Print local-only Terraform plan steps for each workspace.
+##
+_bk_tf_plan_local_steps() {
+  local total_workspaces
+  total_workspaces="$(_bk_tf_total_workspaces)"
+  if [[ "${total_workspaces}" == 0 ]]; then
+    return 0
+  fi
+
+  local workspace queue protected_branches artifact_plugin
+  while IFS=$'\t' read -r workspace queue; do
+    protected_branches="$(_bk_tf_protected_branches_for_workspace "${workspace}")"
+    artifact_plugin="$(_bk_tf_artifacts_plugin "${workspace}" plan)"
+
+    cat << EOF
+- label: ":terraform: Tentative Plan [${workspace}]"
+  branches: "${protected_branches}"
```

Comment:
You can remove this line. Just run the plan local steps regardless of the branch as long as TOOLBOX_PLAN_ONLY=true exists.